### DISCUSSION TITLE
MQTT to Grafana

### DISCUSSION THREAD

**Discussion by raymop** [tagyoureit/nodejs-poolController#533] [REM]
OK, guys, I looked over everything written, and am feeling kinda dumb.  So, I'm asking for help.

I am running njspC, REM and dashPanel.
And also using influx to send njspC data to grafana.
Now I want to use the MQTT broker from REM and display info on a dashboard.
I know that telegraf is used, but cannot (for the life of me) figure how to set that up.
I am not using docker, and already have all the above listed processes up and running.
But, I'm not clear on how to publish topics via mqtt and then have telegraf pick them up for grafana.

Thanks in advance!!!!!

**Comment by tagyoureit** [comment]
We aren't using Telegraph.  Setup a new connection directly to the MQTT broker on the REM homepage:
[EDIT] - make sure to hit the `reset server` button after adding the new connection.
![image](https://user-images.githubusercontent.com/1646840/179372840-55759a6b-5ee7-42e5-8689-098f29dbb78d.png)

Then go to any device and setup a feed and send the values/data you want to any topic.
[EDIT] - the feed should not have a leading slash.  You would user `rem/feed/ph` with my example.  
![image](https://user-images.githubusercontent.com/1646840/179372849-5a4df2f3-f51e-4707-b700-a5aaf352d015.png)
